dune

```diff
@@ -8,3 +8,5 @@
  (mode promote)
  (action
   (copy doc/usage.md %{targets})))
+
+(vendored_dirs stdune)
```

Comment:
I think you'll have to bump the `dune` lang & dependency in opam.

Reply:
It was dune `1.11` already!

Reply:
ah nevermind I saw an example `opam` file in the diff...